src/openingHours.ts

```diff
@@ -104,6 +115,150 @@ export class OpeningHours {
     return this.isOpenOn(new Date());
   }
 
+  /**
+   * Returns next day open
+   */
+  private getNextOpeningDay(date: Date): NextOpeningDay {
+    const today = date.getDay();
+    let times;
+    let i = today;
+
+    for (let key in this.openingHours) {
+      i = i === 7 ? 0 : i + 1;
+      times = this.getTimesOfDay(i);
+      if (times.length > 0) {
+        times = this.openingHours[key];
+        break;
+      }
+    }
+    return {
+      times,
+      nextDay: SIMPLE_DAYS_NAMES[i],
+      isTomorrow: i === today + 1 ? true : false,
+    };
+  }
+
+  private openAt = (formatHours, date: Date, nextDay): string | null => {
```

Comment:
Add a comment above that explain what this function is doing. Same for `openUntil`